C++/Algorithms/Dyanmic Programming/MinimumCount.cpp

```diff
@@ -0,0 +1,29 @@
+int minCount(int n){
+    if(n <= 3){
+        return n;
+    }
+   int *count = new int[n+1];
+   count[0] = 0;
+   count[1] = 1;
+   count[2] = 2;
+   count[3] = 3;
+
+   for(int i = 4; i <= n; i++){
+       int ans = i;
+       for(int j = 1; j <= n/2; j++){
```

Comment:
j <= n/2 is sort of sqrt(n) because squaring of a number greater than n/2 provides a greater number than n. So in order to work with the qualified numbers we reduced it to n/2 or sqrt(n).